example/radolan.py

```diff
@@ -0,0 +1,155 @@
+"""
+=====
+About
+=====
+Example for DWD RADOLAN Composite RW/SF using wetterdienst and wradlib.
+Hourly and gliding 24h sum of radar- and station-based measurements (German).
+
+See also https://docs.wradlib.org/en/stable/notebooks/radolan/radolan_showcase.html.
+
+This program will request daily (RADOLAN SF) data for 2020-09-04T12:00:00
+and plot the outcome with matplotlib.
+
+
+=====
+Setup
+=====
+::
+
+    brew install gdal
+    pip install wradlib
```

Comment:
@gutzbenj said:
> Don't forget to put the wradlib as optional dependency. Also I don't know if we can do anything about GDAL because installing it under Windows is a real struggle.

I would like to keep it like it is for now - just documenting this here. I would like to add it to `cli.py` later and then I will look at bringing it in as an optional dependency.

Reply:
Feel free to add setup guidelines if you can make it work on Windows.